<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_profile_of_manager</class>
 <widget class="QDialog" name="edit_profile_of_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>25</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>690</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>اعمال تغييرات</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_national">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_specialpass">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;رمز ويژه&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_phone">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;تلفن همراه&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;نام&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_lastname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;نام خانوادگي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_usualpass">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;رمز عادي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_national">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;كد ملي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phone">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2_ususal">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3_speci">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4_name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5_lastname">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prevusual">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_prev_usual">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;رمز عادي قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prev_spec">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;رمز ويژه قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prev_spec">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>741</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>1-براي رمز ويژه حداقل 10 كاراكتر و براي رمز عادي حداقل 8 كاراكتر وارد كنيد</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>941</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 85, 127);</string>
   </property>
   <property name="text">
    <string>2- رمز ويژه وارد شده بايد داراي حرف بزرگ ، حرف كوچك ، عدد و ستاره * باشد .</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>3- رمز عادي وارد شده بايستي فقط شامل عدد و كاراكتر باشد.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_name">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;نام قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_lastname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;نام خانوادگي قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_nationalcode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;كد ملي قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_phone">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>IranNastaliq</family>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;تلفن همراه قبلي&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_name_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>32</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_lastname_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>32</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_nationalcode_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>32</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_previous_phone_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>32</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
